src/main/kotlin/org/openmbee/flexo/mms/routes/gsp/ModelRead.kt

```diff
@@ -37,25 +38,34 @@ suspend fun GspLayer1Context<GspReadResponse>.readModel(refType: RefType) {
             when(refType) {
                 RefType.BRANCH -> auth(Permission.READ_BRANCH.scope.id, BRANCH_QUERY_CONDITIONS)
                 RefType.LOCK -> auth(Permission.READ_LOCK.scope.id, LOCK_QUERY_CONDITIONS)
+                RefType.SCRATCH -> auth(Permission.READ_REPO.scope.id, REPO_QUERY_CONDITIONS)
             }
 
-            raw("""
-                graph mor-graph:Metadata {
-                    ${when(refType) {
-                        RefType.BRANCH -> "morb:"
-                        RefType.LOCK -> "morl:"
-                    }} mms:commit/^mms:commit ?ref .
-                    
-                    ?ref mms:snapshot ?modelSnapshot .
-                    
-                    ?modelSnapshot a mms:Model ;
-                        mms:graph ?modelGraph .
-                }
-
-                graph ?modelGraph {
-                    ?s ?p ?o .
+            if(refType == RefType.SCRATCH) {
+                graph("mor-graph:Scratch") {
+                    raw("?s ?p ?o")
                 }
-            """)
+            }
```

Comment:
just fyi will have to deconflict with changes from https://github.com/Open-MBEE/flexo-mms-layer1-service/pull/174 whenever merging into develop